assets/components/spinners/animatedDots.scss

```diff
@@ -0,0 +1,42 @@
+.component-animated-dots {
+
+  margin: 10px auto 0;
+  width: 70px;
+  text-align: center;
+
+  div {
+    width: 18px;
+    height: 18px;
+    background-color: #fff;
+
+    border-radius: 100%;
+    display: inline-block;
+    -webkit-animation: sk-bouncedelay 1.4s infinite ease-in-out both;
```

Comment:
Autoprefixer should [cover](https://github.com/postcss/autoprefixer/wiki/support-list#animation) this.